source/lib/utils/infer-label.ts

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs';
```

Comment:
This will include `fs` module when building for browsers. Would be great to avoid that by providing browser version of this file via package.json's `browser` field.

Reply:
Good point, it came to my mind as well but I totally forgot about this. I was more thinking in line of requiring it after the `if (isNode)` check. What that do the trick?

Reply:
Nope, because bundles scan AST for `require()`, they don't care whether it's unreachable or not.

Reply:
You're right. Although I'm not a big fan of shipping a second browser specific bundle. Could we use something like [`import-lazy`](https://github.com/sindresorhus/import-lazy) to avoid this? Or is this a hacky solution? It somehow is a lazy require right, only if we pass the `if`-check, it should be loaded.

Reply:
> Although I'm not a big fan of shipping a second browser specific bundle. 

But you wouldn't be shipping a browser specific bundle, it would only be a browser specific version of one file (infer-label.ts), see https://github.com/defunctzombie/package-browser-field-spec#replace-specific-files---advanced.

Reply:
Hm yeah, in that case we'll need 2 bundles...

Reply:
I think we can use this to force Webpack to ignore it:

```js
const fuckYouWebpackRequire = typeof __non_webpack_require__ === 'function' ? __non_webpack_require__ : require;
const fs = fuckYouWebpackRequire('fs');
```